### DISCUSSION TITLE
Is there an explanation of how following hashtag works with federation?

### DISCUSSION THREAD

**Discussion by ThomasOwens** [mastodon/mastodon#20579] [Q&A]
I'm trying to piece together how things work.

I understand that the Federated timeline consists of everyone on my instance plus everyone on other instances followed by at least one person on my instance. This seems to imply that not every post is federated to the whole set of instances. My instance doesn't receive all posts from all other instances, but only posts from someone directly followed by someone on my instance. On top of this, the instance moderation rules apply where an instance administrator can disallow posts from certain instances.

However, hashtags add a new dimension. If I follow a hashtag, will I see all of the content from other instances with that hashtag? Or am I limited to people on my instance or people who's posts are federated to my instance (meaning that at least one person on my instance follows them)? My concern is that there may be people out there having a discussion and using one or more hashtags that interest me, but I'll never see those discussions because no one on my instance follows them.

Is following hashtags expanding what gets federated to the instance that I'm on? Or am I perhaps missing whole posts and conversations because no one is following the people who are having the conversation?

**Comment by kevin0t** [comment]
>My concern is that there may be people out there having a discussion and using one or more hashtags that interest me, but I'll never see those discussions because no one on my instance follows them.

Your understanding should be correct. For your instance to federate posts of a user on another server , he/she may have to be followed by someone on your server first to get there post irrespective of you searching them.by hastags.
basically your instance creates a copy of toots from other instances , so on using the search function or lists , it should fetch data from these existing copies.
i am not 100% sure if its true though.

**Comment by OLibutzki** [comment]
I currently run my own personal instance so there is no other account on my instance.

I wonder which consequences it has if I follow a hashtag and I assume: none, because there is no other account which might cause a toot to be replicated into my instance. The only way to get a federated toot into my instance is that an account I follow toots or boosts a toot.

I hope there is some other magic regarding following hashtag...

**Reply by FlominatorTM** [reply]
Maybe https://genealysis.social/@g3rv4@mastodonte.tech helps. Maybe something more will come as reply to https://genealysis.social/@Flominator/109539085735595424
